Client/Frontend/Fakespot/Views/FakespotHighlightsCardView.swift

```diff
@@ -0,0 +1,203 @@
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(highlightStackView)
+        contentView.addSubview(dividerView)
+        contentView.addSubview(moreButton)
+        addSubview(cardContainer)
+
+        NSLayoutConstraint.activate([
+            cardContainer.leadingAnchor.constraint(equalTo: leadingAnchor),
+            cardContainer.topAnchor.constraint(equalTo: topAnchor),
+            cardContainer.trailingAnchor.constraint(equalTo: trailingAnchor),
+            cardContainer.bottomAnchor.constraint(equalTo: bottomAnchor),
+
+            titleLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor,
+                                                constant: UX.contentHorizontalSpace),
+            titleLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: UX.contentTopSpace),
+            titleLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor,
+                                                 constant: -UX.contentHorizontalSpace),
+            titleLabel.bottomAnchor.constraint(equalTo: highlightStackView.topAnchor, constant: -UX.highlightSpacing),
+
+            highlightStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor,
+                                                        constant: UX.contentHorizontalSpace),
+            highlightStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor,
+                                                         constant: -UX.contentHorizontalSpace),
+            highlightStackView.bottomAnchor.constraint(equalTo: dividerView.topAnchor,
+                                                       constant: -UX.highlightStackBottomSpace),
+
+            dividerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            dividerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            dividerView.bottomAnchor.constraint(equalTo: moreButton.topAnchor, constant: -8),
+            dividerView.heightAnchor.constraint(equalToConstant: 1),
+
+            moreButton.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            moreButton.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            moreButton.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+        ])
+    }
+
+    @objc
+    private func showMoreAction() {
+        guard let viewModel else { return }
+
+        isShowingPreview = !isShowingPreview
+        updateHighlights()
+
+        moreButton.setTitle(
+            isShowingPreview ? viewModel.moreButtonTitle : viewModel.lessButtonTitle,
+            for: .normal)
+        moreButton.accessibilityIdentifier = isShowingPreview ? viewModel.moreButtonA11yId : viewModel.lessButtonA11yId
+
+        if !isShowingPreview {
+            recordTelemetry()
+        }
+    }
+
+    private func updateHighlights() {
+        highlightStackView.removeAllArrangedViews()
+
+        for i in 0..<highlightGroups.count {
```

Comment:
forEach? Would be clear we loop all the `highlightGroups`

Reply:
I need the index but I can change it to use `enumerated`.